common-arcana.lic

```diff
@@ -640,4 +640,23 @@ module DRCA
 
     crafting_prepare_spell(training_spells[needs_training], settings)
   end
+
+  def cast_waggle(settings, waggle_sets_name = 'default')
```

Comment:
💡 I don't like 'waggle' and I think it's overall been more confusing than helpful.

Reply:
I think you're right,  and new lichers have no idea what waggles are.  They sound like something irrelevant just by their name.  But if that is their name I'd like to stick to it, unless this is the time we add a transform to dependency that maps wagglesets to a different name and begin the long process of transitioning away from it?  I'd do it, but that's a bigger codebase change. 

Alternately I could make this more generic and accept any hash of spells.

Reply:
I think the only place we have to use 'waggle' is in the setting name itself, since we haven't fixed that yet.

Reply:
I got distracted by other things, but I'm back to this.  Still, I'll look at the wider scale some other time maybe.